src/pages/signin/TermsAndLicenses/index.desktop.js

```diff
@@ -0,0 +1,3 @@
+import TermsOnly from './TermsOnly';
```

Comment:
NAB: Why aren't we including licenses in desktop?

Reply:
It's only a banking requirement on web as explained here: https://github.com/Expensify/Expensify/issues/154672#issue-810980012